clients/python/tests/test_experiments.py

```diff
@@ -1,9 +1,29 @@
+from __future__ import annotations
+
 import json
 import threading
 
 import pytest
 
 from model_registry import ModelRegistry, utils
+from model_registry.types import Artifact, ListOptions, Pager
+
+
+def get_artifacts(
+    model_registry: ModelRegistry,
+    filter_query: str | None = None,
+    artifact_type: str | None = None,
+) -> Pager[Artifact]:
+    def artifacts_list(options: ListOptions) -> list[Artifact]:
+        return model_registry.async_runner(
+            model_registry._api.get_artifacts(
+                filter_query=filter_query,
+                artifact_type=artifact_type,
+                options=options,
+            )
+        )
+
+    return Pager[Artifact](artifacts_list)
```

Comment:
If we have a `get_artifacts` method on the `ModelRegistry` client this is basically what it would be. I'm thinking that this is really a lower-level thing than what we typically have on the client though, so it's here for end-to-end testing of the scenario the front-end will likely do:

Get experiments runs, then get artifacts, and params for those runs:
```
/experiment_runs
/artifacts?artifactType=metric&filterQuery=experimentRunId+IN+(1,2,3)
/artifacts?artifactType=parameters&filterQuery=experimentRunId+IN+(1,2,3)
```